src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php

```diff
@@ -127,21 +130,25 @@ private function display(InputInterface $input, OutputInterface $output, $files)
         }
     }
 
-    private function displayTxt(OutputInterface $output, $filesInfo)
+    private function displayTxt(OutputInterface $stdout, $output, $filesInfo)
     {
         $errors = 0;
 
         foreach ($filesInfo as $info) {
-            if ($info['valid'] && $output->isVerbose()) {
-                $output->writeln('<info>OK</info>'.($info['file'] ? sprintf(' in %s', $info['file']) : ''));
+            if ($info['valid'] && $stdout->isVerbose()) {
```

Comment:
I'm not convinced having the `$stdout` in every methods is needed just in order to use `isVerbose` (`SymfonyStyle` implements `OutputInterface` which has the `getVerbosity` method). However, would it be a good idea to expose those methods in `OutputStyle` ?